security/pkg/server/ca/server.go

```diff
@@ -132,6 +141,22 @@ func extractRootCertExpiryTimestamp(ca CertificateAuthority) float64 {
 	return float64(end.Unix())
 }
 
+// extractCertChainExpiryTimestamp returns the unix timestamp when the cert chain becomes expires.
+func extractCertChainExpiryTimestamp(keyCertBundle util.KeyCertBundle) float64 {
+	cc := keyCertBundle.GetCertChainPem()
+	cert, err := util.ParsePemEncodedCertificate(cc)
+	if err != nil {
+		serverCaLog.Errorf("Failed to parse the cert chain: %v", err)
+		return -1
+	}
+
+	end := cert.NotAfter
+	if end.Before(time.Now()) {
+		serverCaLog.Errorf("Expired Citadel cert chain found, x509.NotAfter %v, please transit your cert chain", end)
```

Comment:
@myidpt should we still use Citadel name in the logging? given istiod is the default mode and the future direction? maye new code can avoid mentioning citadel?

Reply:
I agree. Let's call it Istio CA.